shell/app/electron_main_delegate.cc

```diff
@@ -90,10 +90,13 @@ bool SubprocessNeedsResourceBundle(const std::string& process_type) {
       process_type == ::switches::kZygoteProcess ||
 #endif
 #if defined(OS_MAC)
-      // Mac needs them too for scrollbar related images and for sandbox
-      // profiles.
-      process_type == ::switches::kPpapiPluginProcess ||
+      // Mac needs them for sandbox profiles.
       process_type == ::switches::kGpuProcess ||
+#endif
+#if defined(OS_MAC) || defined(OS_WIN)
+      // Windows needs ppapi for loading pdf metadata, Mac needs it
+      // scrollbar related images
+      process_type == ::switches::kPpapiPluginProcess ||
```

Comment:
Would be good to align this with https://chromium-review.googlesource.com/c/chromium/src/+/2619003/9/chrome/app/chrome_main_delegate.cc

Reply:
Oh, great catch! I'll move ppapi process to all platforms to align with Chrome, will test and make sure everything works as expected 🙂